.github/workflows/docker-publish.yml

```diff
@@ -9,9 +9,12 @@ on:
     # Publish `v1.2.3` tags as releases.
     tags:
       - v*
+  pull_request:
+    branches: [ master ]
```

Comment:
Do you want to create a new image on each PR? or is this for testing this PR?

Reply:
@julio-lopez It won't actually push the image see line 51 ` push: ${{ github.event_name != 'pull_request' }}`

Reply:
But it will try to build the image and validate the Dockerfile